programs/compute-budget/Cargo.toml

```diff
@@ -13,9 +13,19 @@ edition = { workspace = true }
 solana-program-runtime = { workspace = true }
 solana-sdk = { workspace = true }
 
+[dev-dependencies]
+criterion = { workspace = true }
+solana-compute-budget = { workspace = true }
+solana-runtime-transaction = { workspace = true }
```

Comment:
dev-dependent on `solana-runtime-transaction` for bencher to access `process_compute_budget_instructions ()` to process instructions, caused circular dep. Because:
1. `solana-runtime-transaction` dep on `solana-builtins-default-costs` crate to access `MAYBE_BUILTIN_KEY` static map;
2. `solana-builtins-default-costs` dep on `solana-compute-budget-program` (this) crate, in order to get `compute_budget::id()`

Commonly we can solve this by creating a separate `bench-compute-budget-program` crate alone side _this_ crate, but that'd break "structure" set up so far. Foreseeing similar situation might exist for other programs, maybe we adapt this change for all bench projects? (eg instead creating `benches\` within target crate, creating `bench-xyz-program` crate next to it) Or perhaps there are more elegant solution?